### DISCUSSION TITLE
v5 is not faster than v4 when performing secondary packaging

### DISCUSSION THREAD

**Discussion by tantian1498** [webpack/webpack#18477] [Q&A]
The project was migrated from v4 to v5. After testing, it was found that multiple builds without code modification did not speed up the project. In addition, the number of files in node_modules\.cache\webpack increased even though the code did not change. I guess it might be due to an error in hash calculation? How can I troubleshoot the problem?

**Comment by alexander-akait** [comment]
Please make a profile, also try to disable plugins/loaders, without configuration I can't say there is a problem, maybe non official plugins do this, please provide more information
